src/hooks/useOutsideAlerter.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
+
+export default (ref, onClickOutside) => {
+  function handleClickOutside(event) {
```

Comment:
why is this one a funcion and not an arrow as the rest of the code? just checking consistency.

Reply:
I had it this way first,
`export default function useOutsideAlerter(ref, onClickOutside) => {`
`  `  `function handleClickOutside(event) {`
`   ` `...`
`}`
But after checking the others custom hooks, I did the refactor to arrow function, but forgot about the func inside, will refactor now